src/Tasks/AssemblyDependency/ReferenceTable.cs

```diff
@@ -1632,27 +1631,25 @@ List<Exception> exceptions
         )
         {
             MSBuildEventSource.Log.RarComputeClosureStart();
-            {
-                References.Clear();
-                _externallyResolvedPrimaryReferences.Clear();
-                SkippedFindingExternallyResolvedDependencies = false;
+            References.Clear();
+            _externallyResolvedPrimaryReferences.Clear();
+            SkippedFindingExternallyResolvedDependencies = false;
 
-                _remappedAssemblies = remappedAssembliesValue;
-                SetPrimaryItems(referenceAssemblyFiles, referenceAssemblyNames, exceptions);
+            _remappedAssemblies = remappedAssembliesValue;
+            SetPrimaryItems(referenceAssemblyFiles, referenceAssemblyNames, exceptions);
 
-                ComputeClosure();
-            }
-            MSBuildEventSource.Log.RarComputeClosureStop();
+            MSBuildEventSource.Log.RarComputeClosureStop(ComputeClosure());
         }
 
         /// <summary>
         /// Implementation of ComputeClosure.
         /// </summary>
-        private void ComputeClosure()
+        private int ComputeClosure()
```

Comment:
super-nit: Change the name of the method to give a meaning to the return value (e.g. `ComputeClosureAndGetNumberOfResolvedFiles`) or add `<returns>` to the doc comment.